dev-infrastructure/configurations/svc-cluster.bicepparam

```diff
@@ -11,6 +11,7 @@ param disableLocalAuth = false
 param deployFrontendCosmos = false
 param deployMaestroInfra = false
 param maestroNamespace = 'maestro'
+param maxClientSessionsPerAuthName = 1
```

Comment:
Should the default be 2 to cover for pod rollout?

Reply:
updated default value to 2.